src/core/models/aggregate.cc

```diff
@@ -110,9 +110,9 @@ return: Tuple[Frame, Frame]
     number of members per exemplar.
 
     The second element in the tuple is the members frame with the shape of
-    `(frame.nrows, 1)`, each row in this frame corresponds to the
-    row with the same id in the input `frame`. The only column `exemplar_id`
-    has an stype of `int32` and contains the exemplar ids a particular
+    `(frame.nrows, 1)`. Each row in this frame corresponds to the
+    row with the same id in the input `frame`. Only the column `exemplar_id`
+    has an stype of `int32` and contains the exemplar ids that a particular
```

Comment:
Here I wanted to say that we only have one column in this frame. I guess it is better to write it as "The single column `exemplar_id`...".